### PR TITLE
Avoid "Nicosiavägen" in <Beskrivning> getting category:Nicosia

### PR DESCRIPTION
**Pull request by mattiasostmar**
Task: https://phabricator.wikimedia.org/T163669

### CODE REVIEW

File: create_infotexts.py [hunk 1]
```diff
@@ -406,7 +406,8 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
             place_matches = []
             for place in places_mapping:
                 if place in desc_string:
-                    place_matches.append(place)
+                    if not re.search(r"Nicosiavägen", desc_string, re.IGNORECASE):
```
Comment (lokal-profil) [inline create_infotexts.py]: Could you not do this using `if not 'nicosiavägen' in desc_string.lower()`? That way we avoid having to use `re`

**Comment by lokal-profil** [comment]
Should the title of this PR have a "not" somewhere in it?

File: create_infotexts.py (hunk 1 above)
Comment (mattiasostmar) [reply create_infotexts.py]: Done

**Comment by mattiasostmar** [comment]
Altered title for clarification.

**Review by lokal-profil — APPROVED** [review]

**Pull request merged**